src/sinks/pulsar/config.rs

```diff
@@ -82,6 +82,10 @@ pub struct PulsarSinkConfig {
     #[configurable(derived)]
     #[serde(default)]
     pub connection_retry_options: Option<CustomConnectionRetryOptions>,
+
+    #[configurable(derived)]
+    #[serde(default)]
+    pub(crate) tls_options: Option<PulsarTlsOptions>,
```

Comment:
```suggestion
    pub(crate) tls: Option<PulsarTlsOptions>,
```

Apologies, missed this one before. We should call this `tls` to match the other components.

Reply:
fixed in https://github.com/vectordotdev/vector/pull/22148/commits/59f063490af1522516515056008c34245ac9b078